apps/src/templates/sectionProgress/progressTables/ProgressTableContainer.jsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import PropTypes from 'prop-types';
+import i18n from '@cdo/locale';
+import {scriptDataPropType} from '../sectionProgressConstants';
+import {studentLevelProgressType} from '@cdo/apps/templates/progress/progressTypes';
+import {sectionDataPropType} from '@cdo/apps/redux/sectionDataRedux';
+import {getCurrentScriptData} from '@cdo/apps/templates/sectionProgress/sectionProgressRedux';
+import styleConstants from '@cdo/apps/styleConstants';
+import ProgressTableStudentList from './ProgressTableStudentList';
+import ProgressTableContentView from './ProgressTableContentView';
+
+/**
+ * Since our progress tables are built out of standard HTML table elements,
+ * we can leverage CSS classes for laying out and styling those elements.
+ */
+import progressTableStyles from './progressTableStyles.scss';
+
+const styles = {
+  container: {
+    width: styleConstants['content-width']
+  },
+  studentList: {
+    display: 'inline-block',
+    verticalAlign: 'top'
+  },
+  contentView: {
+    display: 'inline-block',
+    width: parseInt(progressTableStyles.CONTENT_VIEW_WIDTH)
+  }
+};
+
+class ProgressTableContainer extends React.Component {
```

Comment:
(minor) I think a comment here about the purpose of this component would be helpful.  Is it mainly to synchronize the scrolling between the student list and the bubbles?

Reply:
i want to rename this component. the abstraction should be "i am a section progress table, just tell me how to render lesson cells" -- which i don't think is implicit with this name. the underlying implementation of needing to synchronize two tables shouldn't be relevant to anyone wanting to render a section progress table. unless i hear any better suggestions, i'm going to update this to `SectionProgressTable` (and prepend all the child components with `Section` as well). i meant to add a comment to each of these components before opening PRs (similar to what i wrote up in the tech spec), but forgot to do so before handing off to Maureen. i will do so while renaming.